NEWS.md

```diff
@@ -0,0 +1,3 @@
+# baselinenowcast 0.0.0.100
```

Comment:
dev version convention is usually a number in the 1000s. Do I know why? No I don't

Reply:
Ok so like 0.0.0.1000??